src/ln/channel.rs

```diff
@@ -2345,9 +2349,10 @@ impl Channel {
 	/// Indicates that the latest ChannelMonitor update has been committed by the client
 	/// successfully and we should restore normal operation. Returns messages which should be sent
 	/// to the remote side.
-	pub fn monitor_updating_restored(&mut self) -> (Option<msgs::RevokeAndACK>, Option<msgs::CommitmentUpdate>, RAACommitmentOrder, Vec<(PendingForwardHTLCInfo, u64)>, Vec<(HTLCSource, PaymentHash, HTLCFailReason)>) {
+	pub fn monitor_updating_restored(&mut self) -> (Option<msgs::RevokeAndACK>, Option<msgs::CommitmentUpdate>, RAACommitmentOrder, Vec<(PendingForwardHTLCInfo, u64)>, Vec<(HTLCSource, PaymentHash, HTLCFailReason)>, bool) {
```

Comment:
Hmm I was reading again resend_order comment, which maybe should be refer by function comment, and "Note that because there can only ever be one in-flight CS and/or one in-flight RAA at any time..." is right but really obscure.
IIRC already commented on this comment, but a clearer could be "Protocol update going forward by committing to a new state then revoking older one by local peer, follow by the same sequence again while inverting jobs , when remote peer sends us CS we have to reply by RAA (so we resend_order so), when it sends us RAA, we have to reply CS.

Reply:
Hmm, I'm not quite sure what your objection is?

Reply:
Just a nit, was thinking aloud about a better resend_order comment now that I understand a bit better channel_reestablish logic

Reply:
Hmm, happy to think about it in another pr, but didn't touch it here.